tests/analysis/chemenv/coordination_environments/test_structure_environments.py

```diff
@@ -151,8 +149,6 @@ def test_light_structure_environments(self):
         assert len(nb_set) == 4
         assert hash(nb_set) == 4
 
-        assert nb_set == nb_set
-
         assert (
```

Comment:
better to keep these as well though they should better be in separate test methods `def test_eq`, `test_equal` (the name of `__eq__` test methods is another thing to standardize across the code base)